docs/src/debugging.rst

```diff
@@ -16,3 +19,263 @@ In order to use this for JSON views, you have to append ``?debug`` to the URL, e
+   * The third box on the left is also empty. It will list currently existing processes.
+   * The bottom box on the left allows you to write custom (python) expressions and to see their value. This is an extremely powerful feature.
+     Simply enter insert mode, type the python expression, and press enter. For more, see `this section <https://github.com/rcarriga/nvim-dap-ui?tab=readme-ov-file#watch-expressions>`_ in the docs.
+
+#. Navigate to the ``get`` method and place your cursor at the first line of interest within it. Use the "toggle breakpoint" key bind or the ``DapToggleBreakpoint`` command to set a breakpoint.
+   The breakpoint appears in the second box on the left.
+
+   .. image:: images/debugging/debug-nvim-03-breakpoint.png
+      :alt: Add a breakpoint
+
+#. In your browser, navigate to any page form view, for example: "Willkommen" in "Stadt Augsburg".
+   Note that your browser will appear to be stuck in the page list view, unable to finish loading the page form view - this is intentional!
+   The red breakpoint indicator has changed to a gray arrow, indicating the line the debugger is currently stopped at.
+   The information in the boxes on the left have changed to reflect the program state at this line.
+   You can navigate through the boxes with your usual key binds and extend the variable definitions to see their current values.
+
+   .. image:: images/debugging/debug-nvim-04-start-debugging.png
+      :alt: First debugging step
+
+#. Pressing the (now no longer grayed-out) "step over" button, the cursor will move one line down.
+   At the same time, the variable ``region`` and its value is added to the "Locals" section in the top-left box, since it has been created in the execution step we just performed.
+   If you did install the ``nvim-dap-virtual-text`` plugin, variable values will also be placed next to that variable's definition in the code, where it will be updated when the value changes.
+   This can be a more intuitive visualization than using the variable box in the top-left corner.
+
+   .. image:: images/debugging/debug-nvim-05-step-over.png
+      :alt: Step over demonstration
+
+#. Continue pressing "step over" or try one of the other stepping mechanisms until you have found the information you are looking for.
+   Note that the debugging process will never stop if you simply keep clicking "step over", even after you have reached and moved past the ``return`` statement at the bottom of the ``get`` method.
+#. If you want to end the debugging process, click the "play" button to let the CMS run normally again.
+   Your debugging breakpoint will still exist, so repeating the attempt to load a page form view will put you right back into the debugging process.
+
+
+
+Nixvim
+^^^^^^
+
+If you are a user of the `nixvim <https://github.com/nix-community/nixvim>`_ project (there's dozens of us! Dozens!!), include the following snippet somewhere inside your nix config.
```

Comment:
>  (there's dozens of us! Dozens!!)

:laughing:

Reply:
😄 
Should I remove it? ^^

Reply:
No, not for me. I don't mind a little humor here and there :laughing: